src/patternfly/components/ProgressStepper/examples/ProgressStepper.md

```diff
@@ -0,0 +1,368 @@
+    {{> progress-stepper-step-icon}}
+    {{#> progress-stepper-step-main}}
+      {{#> progress-stepper-step-title}}
+        In process
+      {{/progress-stepper-step-title}}
+    {{/progress-stepper-step-main}}
+  {{/progress-stepper-step}}
+  {{#> progress-stepper-step progress-stepper-step--IsPendingAlt="true"}}
+    {{> progress-stepper-step-icon}}
+    {{#> progress-stepper-step-main}}
+      {{#> progress-stepper-step-title}}
+        Pending
+      {{/progress-stepper-step-title}}
+    {{/progress-stepper-step-main}}
+  {{/progress-stepper-step}}
+{{/progress-stepper}}
+```
+
+
+## Documentation
+
+### Overview
+The progress stepper is intended to show progress through a finite number of discrete steps.
+
+Add a modifier class to the progress stepper to change the orientation or alignment: `.pf-m-center`, `.pf-m-vertical`, or `.pf-m-compact`.
+
+Steps can be modified with `.pf-m-success`, `.pf-m-warning`, `.pf-m-danger`, and `.pf-m-info` to change their color. Use modifiers `.pf-m-pending` and `.pf-m-current` to indicate progress through the steps.
+### Accessibility
+| Attribute | Applied to | Outcome |
+| -- | -- | -- |
+| `aria-label="Completed step"` | `.pf-c-progress-stepper__step` |  Indicates the step is complete. |
+| `aria-label="Current step"` | `.pf-c-progress-stepper__step` |  Indicates the current step. |
+| `aria-label="Pending step"` | `.pf-c-progress-stepper__step` |  Indicates the step is pending. |
+| `aria-label="In process step"` | `.pf-c-progress-stepper__step` |  Indicates the step is in process. |
+| `aria-label="Step with issue"` | `.pf-c-progress-stepper__step` |  Indicates the step has a warning. |
+| `aria-label="Step with failure"` | `.pf-c-progress-stepper__step` |  Indicates the step has a failure. |
+| `aria-label="Information alert"` | `.pf-c-progress-stepper__step` |  Indicates the information alert. |
+| `aria-hidden="true"` | `.pf-c-progress-stepper__step-icon <i>` |  Hides icon for assistive technologies. **Required** |
+
+### Usage
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `.pf-c-progress-stepper` | `<ol>` | Applies default progress stepper styling.  **Required** |
+| `.pf-c-progress-stepper__step` | `<li>` | Defines each step of the progress stepper. **Required** |
+| `.progress-stepper-step-connector` | `<div>` | Creates the connecting line between steps **Required** |
+| `.progress-stepper-step-icon` | `<span>` | Creates the step node and contains the icon designating the type of step. This element is required, but may be empty if no icon is used for the step. **Required** |
+| `.progress-stepper-step-main` | `<div>` | Contains the main text content of the step. This element is required, but may be empty if title and description are not used. **Required** |
+| `.progress-stepper-step-title` | `<div>` | Contains the title of the step. |
+| `.progress-stepper-step-description` | `<div>` | Contains the description of the step. |
+
+### Modifiers
```

Comment:
I'm not against adding modifiers in their own table, but all of the other components have them in the usage table.

Reply:
We have a few with a separate section; personally I like it but removed it here for now. Maybe it warrants a further look across all components.